v3/1upkeyboards/pi40/pi40.json

```diff
@@ -0,0 +1,166 @@
+{
+    "name": "pi40",
+    "vendorId": "0x6F75",
+    "productId": "0x5600",
+    "keycodes": ["via/keycodes", "via/qmk_lighting"],
+    "menus": ["qmk_rgb_matrix"],
```

Comment:
Please remove these for V2 definitions. It is not currently supported yet until V3 is merged in. https://github.com/qmk/qmk_firmware/pull/18222

Reply:
I moved the folder and file the src directory, and removed those lines of code you requested.